helm-chart/utils/postgres_migrations/version_upgrades/README.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading postgresql
```

Comment:
we could add a comment to clarify that these instructions are not specific to Renku but just a guideline on how we (will) upgrade the PostgreSQL version in our deployments, otherwise people are welcome to use their own methods as well.

Reply:
done